packages/api/test/indexer.test.ts

```diff
@@ -527,18 +527,51 @@ describe("Indexer router", async () => {
         expect(transactionForks).toEqual(expectedTransactionForks);
       });
 
-      it("should not mark any block when current head slot is given", async () => {
-        await authorizedCaller.indexer.handleReorgedSlot({ newHeadSlot: 108 });
+      it("should return the number of updated slots", async () => {
+        const result = await authorizedCaller.indexer.handleReorgedSlots(input);
 
-        const transactionForks =
-          await authorizedContext.prisma.transactionFork.findMany();
+        expect(result).toEqual({
+          totalUpdatedSlots: input.reorgedSlots.length,
+        });
+      });
+
+      it("should ignore non-existent slots and mark the ones that exist as reorged", async () => {
+        const reorgedSlots = [106, 107, 99999];
```

Comment:
Maybe we can also test a request of non-existent slots alone as well?

Reply:
Sure